samza-kafka/src/main/java/org/apache/samza/system/kafka/KafkaSystemAdmin.java

```diff
@@ -541,8 +541,8 @@ public KafkaStreamSpec toKafkaSpec(StreamSpec spec) {
           new KafkaStreamSpec(spec.getId(), spec.getPhysicalName(), systemName, 1, coordinatorStreamReplicationFactor,
               coordinatorStreamProperties);
     } else if (spec.isCheckpointStream()) {
-      kafkaSpec = KafkaStreamSpec.fromSpec(StreamSpec.createCheckpointStreamSpec(spec.getPhysicalName(), systemName))
-              .copyWithReplicationFactor(Integer.parseInt(new KafkaConfig(config).getCheckpointReplicationFactor().get()));
+      kafkaSpec = KafkaStreamSpec.fromSpec(spec)
```

Comment:
Minor: Fix indentation.

Reply:
Here's how this is done in KafkaCheckpointManager:
```
    val checkpointSpec = KafkaStreamSpec.fromSpec(StreamSpec.createCheckpointStreamSpec(checkpointTopic, checkpointSystemName))
        .copyWithReplicationFactor(kafkaConfig.getCheckpointReplicationFactor.get.toInt)
        .copyWithProperties(kafkaConfig.getCheckpointTopicProperties)
```

Can we do that here for consistency?

Reply:
1. Fixed the indentation. 
2. Updated the checkpoint spec building to be consistent with `KafkaCheckpointManager`.